docs/gambit/mutation-verifier.md

```diff
@@ -35,129 +34,205 @@ If you are on Linux, you will need pip version 20.3 or above.
 
-```
-certoraMutate --prover_conf path/to/prover/prover.conf --mutation_conf path/to/mutation/mutation.conf
+```sh
+certoraMutate --prover_conf path/to/prover.conf --mutation_conf path/to/mutation.mconf
 ```
 
+If you do, the script will generate code mutants, 
+  then submit a verification job per mutant to Certora's server.
+
 ```{note}
 You must run `certoraMutate` from the root of the Solidity project directory.
-The files `prover.conf` and `mutation.conf`
-can be in their own directories, but must always be within the project directory.
+The files `prover.conf` and `mutation.conf` can be in their own directories, 
+  but must always be within the project directory.
 All paths in `mutation.conf` are relative to the parent directory containing `mutation.conf`.
-Paths in `prover.conf` are all relative to the project directory's root, which is assumed to be the working directory.
+Paths in `prover.conf` are all relative to the project directory's root, 
+  which is assumed to be the working directory.
 ```
 
-## Configurations
-The tool expects two separate configuration files:
-the configuration file which defines the execution of mutant generation (`--mutation_conf`),
-and the configuration file which defines execution of the Prover (`--prover_conf`).
-Here is a simple configuration file setup using the example above:
-
-In `prover.conf`:
+When it finishes successfully, you should see the following lines printed:
 
-```json
-{
-  "files": [
-    "C.sol"
-  ],
-  "verify": "C:c.spec"
-}
 ```
-In `mutation.conf`:
+********************  PROVER END  ********************
+You will receive an email notification when this mutation test is completed. It may take several hours.
+```
+
+Verifying all the mutants is a heavy computational process that may take several hours.
+Once it is completed, you should receive an email that looks like this:
+
+![Mutation suceeded email](doc/email_mutation_success.png)
+
+
+## Mutation Configuration
+The tool expects a configuration file which defines the execution of 
+  mutant generation (`--mutation_conf`).
+Here is a simple configuration file setup using the example above in `mutation.mconf`:
 
 ```json
-{ "gambit": [{
+{ 
+  "gambit": [{
     "filename" : "C.sol",
     "num_mutants": 5
   }]
 }
 ```
 
+### Automatic Generation
+You can generate a mutation configuration automatically from a Prover configuration 
+  with the flag `--generate_mutation_conf`.
+The resulting configuration may need some manual path fixing or the addition of other
+ `certoraMutate` specific options.
+
+Example usage:
+```sh
+certoraRun path/to/prover.conf --generate_mutation_conf path/to/generated/mutation.mconf
+```
+
+
+
 ### Manual Mutations
-You can add manual mutations to `mutation.conf` like so:
+
+You have the option to include manually generated mutated files, 
+  not produced by Gambit, in your mutation test. 
+We refer to these as 'manual mutations.' 
+They can be beneficial for evaluating verification coverage related to 
```

Comment:
"specific changes that must always be checked," not so clear to me what are the "specific changes"
Can we say that in general manual mutation is for "regression testing"

Reply:
Great feedback